web/angular.json

```diff
@@ -11,6 +11,7 @@
         "build": {
           "builder": "@angular-builders/custom-webpack:browser",
           "options": {
+            "allowedCommonJsDependencies": ["chart.js"],
```

Comment:
Possible purge of `chart.js` dependency in favor of a native implementation during later rewriting process